src/java.base/share/classes/java/io/Console.java

```diff
@@ -117,13 +117,13 @@ public PrintWriter writer() {
      * This method is intended to be used by sophisticated applications, for
      * example, a {@link java.util.Scanner} object which utilizes the rich
      * parsing/scanning functionality provided by the {@code Scanner}:
-     * <blockquote><pre>
-     * Console con = System.console();
-     * if (con != null) {
-     *     Scanner sc = new Scanner(con.reader());
-     *     ...
+     * {@snippet lang=java :
+     *         Console con = System.console();
+     *         if (con != null) {
+     *             Scanner sc = new Scanner(con.reader());
+     *             ...
```

Comment:
I'm not sure how you feel about this, but when I have been converting `<pre>` blocks to `@snippet`, I usually try to get rid of non code. Since snippet allows user's to copy the example code, I feel like it's redudant to leave in non code, since they will likely delete it anyways.

Reply:
Agree, that I would make the code valid when using a snippet

Reply:
There's a replacement tag for this. Some people use label, e.g.
```
:code // @replace substring=":code" replacement="..."
```

This way, the snippet remains compilable and rendered the same as before.